__tests__/app.test.js

```diff
@@ -29,3 +29,35 @@ describe("/api/categories", () => {
     });
   });
 });
+describe("/api/reviews/:review_id", () => {
+  describe("GET", () => {
+    test("200: should return a review object with following properties: review_id, title, review_body, designer, review_img_url, votes, category (references slug in categories), owner (references users primary key i.e. username), created_at ", () => {
+      return request(app)
+        .get("/api/reviews/6")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toBeInstanceOf(Object);
+          expect(body.review).toHaveProperty("review_id", expect.any(Number));
+          expect(body.review).toHaveProperty("title", expect.any(String));
+          expect(body.review).toHaveProperty("review_body", expect.any(String));
```

Comment:
in this test, we are expecting the specific values associated with the 6th review in the list, NOT any old values. Need to change this to be more accurate otherwise this test isn't satisfactory.